languages/js/src/dataFiltering.test.ts

```diff
@@ -606,7 +688,8 @@ resource Issue {
   "read" if "reader" on "parent";
 }
 
-has_relation(repo: Repo, "parent", issue: Issue) if repo = issue.repo;
+#has_relation(repo: Repo, "parent", issue: Issue) if issue.repo = repo;
+has_relation(repo: Repo, "parent", _issue: Issue{repo:repo});
```

Comment:
Delete one of these two? I assume the former matches the other language tests and the latter tests the bug you fixed in this PR?